### ISSUE TITLE
Creating property from column alias

### ISSUE THREAD

**Issue by Topper865** [agrosner/DBFlow#1522]
`Select(Item_Table.id.withTable(), Item_Table.title.withTable(), Item_Table.item_code.withTable()
                , Item_Table.item_barcode.withTable(), sum(StockLedger_Table.quantity_in.withTable()).`as`("totalUnitsIn"),
                sum(StockLedger_Table.quantity_out.withTable()).`as`("totalUnitsOut"),
                sum(StockLedger_Table.price_in.withTable()).`as`("totalPriceIn"), sum(StockLedger_Table.price_out.withTable())
                .`as`("totalPriceOut"), sum(StockLedger_Table.cost_in.withTable()).`as`("totalCostIn"),
                sum(StockLedger_Table.cost_out.withTable()).`as`("totalCostOut"), sum(StockLedger_Table.quantity_in.withTable()
                .minus(StockLedger_Table.quantity_out.withTable())).`as`("currentStock")`

I want to use columns created using sum operator as a property with alias like `totalPriceIn` . Can I acheive this in any way?

**Comment by agrosner** [comment]
not sure I understand exactly what you mean. Maybe a further example? (sorry for late response!)

**Comment by Topper865** [comment]
Hello @agrosner , Thanks for getting back to me,
Its long time now I worked on project, I don't remember exactly but I solved my problem using StringQuery. Actually , I wanted to calculate total profit by using columns created in query. For example `(totalPriceOut - totalPriceIn) as profitEarned` . In DBFlow we can use predefined columns as property but dynamically created columns can't be used as property.
